instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './search.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Instagram from './instagram.svg';
+// import PropTypes from "prop-types";
+
+const SearchBar = (props) => {
+    return (
+        <section className="search-bar">
+            <section className="insta-logo">
+                <img src={ Instagram } alt="instagram logo"></img>
+                <p>|</p>
```

Comment:
did you consider using a border instead of `|`?

Reply:
I did, but I was running out of time and didn't have time to go back and change that.